src/molecules/Select/Input.test.tsx

```diff
@@ -8,14 +8,16 @@ const mockClick = jest.fn();
 const setup = (
   isOpen: boolean = false,
   placeholder: string = 'I am a placeholder',
-  readonly: boolean = false
+  readonly: boolean = false,
+  dataQaId: string | undefined = undefined
```

Comment:
What's the last part of this doing?

Reply:
Ha, well the intention was to set it to `undefined` by default, but I may have gone about that in a weird way :-).